src/views/checkups/self-validation/components/actions/CheckupsSelfValidationActionFactory.tsx

```diff
@@ -39,7 +38,12 @@ export const CheckupsSelfValidationActionFactory = {
       deleteSelfValidationCheckup(configMap, jobs).catch((err) => {
         kubevirtConsole.error('Failed to delete self-validation checkup:', err);
       });
-      navigate(`/k8s/ns/${getNamespace(configMap)}/checkups/${CHECKUP_URLS.SELF_VALIDATION}`);
+
+      const newPath = location.pathname.endsWith(getName(configMap))
+        ? location.pathname.replace(getName(configMap), '')
```

Comment:
looks like a variant of `trimLastHistoryPath()` (which we have at least 2 implementations already)